vtex/loaders/orders/order.ts

```diff
@@ -0,0 +1,38 @@
+import { RequestURLParam } from "../../../website/functions/requestToParam.ts";
+import { AppContext } from "../../mod.ts";
+import { Userorderdetails } from "../../utils/openapi/vcs.openapi.gen.ts";
+import { parseCookie } from "../../utils/vtexId.ts";
+
+export interface Props {
+  slug: RequestURLParam;
+}
+
+export default async function loader(
+  props: Props,
+  req: Request,
+  ctx: AppContext,
+): Promise<Userorderdetails | null> {
+  const { vcs } = ctx;
+  const { cookie, payload } = parseCookie(req.headers, ctx.account);
+
+  const { slug } = props;
+
```

Comment:
I am not sure about that. We could verify if user is logged in. But somehow, the VTEX can show orderPlaced page without user logged in.